[
    {
        "cve": "CVE-2024-30251",
        "created_datetime": "2024-05-13T01:47:22",
        "author_id": "22899183",
        "data_id": "1789834793590374775",
        "reposts": 1,
        "post": "rt @iototsecnews: aiohttp \u306e dos \u8106\u5f31\u6027 cve-2024-30251 \u304c fix\uff1a\u305f\u3060\u3061\u306b\u30a2\u30c3\u30d7\u30c7\u30fc\u30c8\u3092\uff01\nhttps://t.co/d7iihy61gj\n#aiohttp #asynchronoushttp #cyberattack #ddos\u2026",
        "audience": 3554,
        "audience_total": 3554,
        "total_reposts": 1,
        "post_count": 0,
        "author": "agile cat",
        "username": "agile_cat",
        "v_score": 0.702973,
        "e_score": null,
        "base_score": "7.5",
        "description": "AIOHTTP IS AN ASYNCHRONOUS HTTP CLIENT/SERVER FRAMEWORK FOR ASYNCIO AND PYTHON. IN AFFECTED VERSIONS AN ATTACKER CAN SEND A SPECIALLY CRAFTED POST (MULTIPART/FORM-DATA) REQUEST. WHEN THE AIOHTTP SERVER PROCESSES IT, THE SERVER WILL ENTER AN INFINITE LOOP AND BE UNABLE TO PROCESS ANY FURTHER REQUESTS. AN ATTACKER CAN STOP THE APPLICATION FROM SERVING REQUESTS AFTER SENDING A SINGLE REQUEST. THIS ISSUE HAS BEEN ADDRESSED IN VERSION 3.9.4. USERS ARE ADVISED TO UPGRADE. USERS UNABLE TO UPGRADE MAY MANUALLY APPLY A PATCH TO THEIR SYSTEMS. PLEASE SEE THE LINKED GHSA FOR INSTRUCTIONS.",
        "last_modified_datetime": "2024-05-02 18:00:00",
        "published_datetime": "2024-05-02 14:15:00",
        "vendor_logo": null,
        "profile_url": null,
        "feed_name": "twitter"
    }
]